README.md

```diff
@@ -1,63 +1,121 @@
-# Codacy Staticcheck
+# codacy-staticcheck
 
-This is the docker engine we use at Codacy to have [Staticcheck](https://github.com/dominikh/go-tools/tree/master/cmd/staticcheck) support.
-You can also create a docker to integrate the tool and language of your choice!
-Check the **Docs** section for more information.
+A standalone tool that converts [staticcheck](https://staticcheck.io/)
+diagnostics to Codacy's format.
 
-[![Codacy Badge](https://api.codacy.com/project/badge/Grade/a4ec667e01894a4890d561968428c828)](https://www.codacy.com/gh/codacy/codacy-staticcheck?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=codacy/codacy-staticcheck&amp;utm_campaign=Badge_Grade)
-[![CircleCI](https://circleci.com/gh/codacy/codacy-staticcheck.svg?style=shield&circle-token=:circle-token)](https://circleci.com/gh/codacy/codacy-staticcheck)
+It allows running staticcheck either locally or as part of your CI process and then integrating the results into your Codacy workflow. This way, Codacy will present the results coming from staticcheck alongside all the other code quality information in the dashboards.
```

Comment:
Maybe we can link to our client-side tools documentation. https://docs.codacy.com/related-tools/client-side-tools/
Also, we should update that page when we add the tool officially